src/Media/ImageUploaderService.php

```diff
@@ -105,12 +105,12 @@ private function guardFileSizeLimit(UploadedFile $file): void
         $fileSize = filesize($filePath);
 
         if ($this->maxFileSize && !$fileSize) {
-            throw new ImageSizeError('There is a maximum size and we could not determine the size of the uploaded image.');
+            throw new InvalidFileSize('There is a maximum size and we could not determine the size of the uploaded image.');
```

Comment:
I think making it a `RuntimeException` makes more sense indeed. If something like this would happen, we most likely have an infra or config issue and not a user error.